### PR TITLE
Update svg2pdf.js

### PR DESCRIPTION
**Pull request by Anderman**
Fix rotation. see https://github.com/yWorks/svg2pdf.js/issues/120

### CODE REVIEW

**Comment by yGuy** [comment]
Thank you for the report and also for the proposed fix, which at the first glance LGTM. Could you add a minimal test-case to the pull-request, too, please? That would allow us to release the fix more quickly. Thanks again!

**Comment by Anderman** [comment]
@yGuy,

I added a test. Every looks ok to me.

File: tests/runTests.js [hunk 1]
```diff
@@ -1,51 +1,52 @@
-const debug = false;
+const debug = true;
```
Comment (HackbrettXXX) [inline tests/runTests.js]: Set back to `false`

File: tests/runTests.js [hunk 2]
```diff
@@ -1,51 +1,52 @@
...
 
 const tests = [
-    "attribute-style-precedence",
```
Comment (HackbrettXXX) [inline tests/runTests.js]: Please only add the test case and revert the whitespace changes.

File: tests/highchart-wordcloud/spec.svg [hunk 3]
```diff
@@ -0,0 +1,22 @@
...h Highcharts 7.2.1</desc><defs><clipPath id="highcharts-8hytlhj-208-"><rect x="0" y="0" width="841" height="311" fill="none"></rect></clipPath></defs><rect fill="rgba(255,255,255,0.6)" class="highcharts-background" x="0" y="0" width="860.520720124746" height="400" rx="0" ry="0"></rect><rect fill="none" class="highcharts-plot-background" x="10" y="74" width="841" height="311"></rect><g class="highcharts-grid highcharts-xaxis-grid" data-z-index="1"></g><g class="highcharts-grid highcharts-yaxis-grid" data-z-index="1"></g><rect fill="none" class="highcharts-plot-border" data-z-index="1" x="10" y="74" width="841" height="311"></rect><g class="highcharts-axis highcharts-xaxis" data-z-index="2"><path fill="none" class="highcharts-axis-line" data-z-index="7" d="M 10 385 L 851 385"></path></g><g class="highcharts-axis highcharts-yaxis" data-z-index="2"><path fill="none" class="highcharts-axis-line" data-z-index="7" d="M 10 74 L 10 385"></path></g><g class="highcharts-series-group" data-z-index="3"><g data-z-index="0.1" class="highcharts-series highcharts-series-0 highcharts-wordcloud-series highcharts-tracker" transform="translate(430.5,229.5) scale(3.9196534068530915 3.9196534068530915)">
```
Comment (HackbrettXXX) [inline tests/highchart-wordcloud/spec.svg]: Please reduce the test case to 4 cases
* `0,0`
* `x,0`
* `0,y`
* `x.y`
...and get rid of all other properties (style, opacity, ...)
